lib/components/Bullet/Bullet.tsx

```diff
@@ -32,32 +32,32 @@ export const Bullet = ({ children }: BulletProps) => {
     <TextContext.Provider value={textContextValue}>
       <Box
         component={component}
-        className={[
-          useText({
+        {...useStackItem({
+          space,
+          align: 'left',
+        })}
+      >
+        <Box
+          className={useText({
             size,
             baseline: true,
             tone,
-          }),
-          useStackItem({
-            component,
-            space,
-            align: 'left',
-          }),
-        ]}
-      >
-        <Box display="flex">
-          <Box
-            display="flex"
-            alignItems="center"
-            className={useLineHeightContainer(size)}
-          >
+          })}
+        >
```

Comment:
Now that we just have the text styles on it's own node, we should be able to use a `Text` component here